libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/Flyout.tsx

```diff
@@ -111,175 +119,177 @@ export class Flyout extends React.Component<IFlyoutProps, IFlyoutState> {
-                  <Image
-                    src={`data:image/jpg;base64,${this.props.explanations.get(
-                      index
-                    )}`}
-                    width="700px"
-                    style={{ position: "relative", right: 85 }}
-                  />
+                  <Text variant="large" className={classNames.title}>
+                    {localization.InterpretVision.Dashboard.panelInformation}
+                  </Text>
                 </Stack.Item>
-              ) : (
-                <Stack.Item>
-                  <Spinner
-                    label={`${localization.InterpretVision.Dashboard.loading} ${item?.index}`}
+                <Stack.Item className={classNames.featureListContainer}>
+                  <List
+                    items={this.state.metadata}
+                    onRenderCell={onRenderCell}
                   />
                 </Stack.Item>
-              )}
+              </Stack>
             </Stack>
-            <Stack.Item>
-              <Separator className={classNames.separator} />
-            </Stack.Item>
-            <Stack
-              tokens={{ childrenGap: "l2" }}
-              className={classNames.sectionIndent}
-            >
+            <Stack>
+              <Stack.Item>
+                <Separator className={classNames.separator} />
+              </Stack.Item>
               <Stack.Item>
                 <Text variant="large" className={classNames.title}>
-                  {localization.InterpretVision.Dashboard.panelInformation}
+                  {localization.InterpretVision.Dashboard.panelExplanation}
                 </Text>
               </Stack.Item>
-              <Stack.Item className={classNames.featureListContainer}>
-                <List
-                  items={this.state.metadata}
-                  onRenderCell={this.onRenderCell}
-                />
-              </Stack.Item>
+              <Stack>
+                {
+                  <ComboBox
+                    id="objectSelection"
+                    label={"Choose a detected object"}
+                    onChange={this.selectODChoiceFromDropdown}
+                    selectedKey={this.state.odSelectedKey}
+                    options={this.state.selectableObjectIndexes}
+                    className={"classNames.dropdown"}
+                    styles={FluentUIStyles.smallDropdownStyle}
+                  />
+                }
+                <Stack>
+                  {!this.props.loadingExplanation[item.index][
+                    +this.state.odSelectedKey.slice(7)
+                  ] ? (
+                    <Stack.Item>
+                      <Image
+                        src={`data:image/jpg;base64,${this.props.explanations
+                          .get(item.index)
+                          ?.get(+this.state.odSelectedKey.slice(7))}`}
+                        width="700px"
+                        style={{ position: "relative", right: 85 }}
```

Comment:
move to styles file?

                        width="700px"
                        style={{ position: "relative", right: 85 }}

Reply:
Fixed, hopefully this is more clear. Not super familiar with typescript, so let me know if there is a more simplistic way of importing style types :)